src/check_ast.rs

```diff
@@ -1051,6 +1051,10 @@ where
                     pyupgrade::plugins::super_call_with_parameters(self, expr, func, args);
                 }
 
+                if self.settings.enabled.contains(&CheckCode::U012) {
+                    pyupgrade::plugins::unnecessary_encode_utf8(self, expr, func, args);
```

Comment:
I think this is missing an argument (`locator`), but maybe it should just be omitted, and then you can access `checker.locator` in `unnecessary_encode_utf8`?

Reply:
yep i forgot to commit this part alongside

Reply:
All good!